duration.go

```diff
@@ -47,6 +49,10 @@ var (
 // Parse attempts to parse the given duration string into a *Duration,
 // if parsing fails an error is returned instead
 func Parse(d string) (*Duration, error) {
+	if !strings.Contains(d, "P") {
```

Comment:
I'm sorry, I see a problem here.

#4 opened by @jrdouceur was expressly requesting for a leafing P which is in the RFC

Here you are looking for P character, not a leading P

You should have used [strings.HasPrefix](strings)

Reply:
Ah yeah good point. I'll fix it up.

Reply:
Thanks, fixed with 514a8675f71110de9d39ae098be2b119b154dbf9 in #6

Reply:
Not actually fixed haha. I'm about to fix it again.